corehq/apps/accounting/forms.py

```diff
@@ -1993,77 +1996,39 @@ def send_message(self):
 
     def __init__(self, domain, web_user, on_annual_plan, data=None, *args, **kwargs):
-        self.domain = domain
-        self.web_user = web_user
-        super(AnnualPlanContactForm, self).__init__(data, *args, **kwargs)
-        from corehq.apps.domain.views.accounting import (
-            DomainSubscriptionView,
-            SelectPlanView,
-        )
-        self.helper = FormHelper()
-        self.helper.label_class = 'col-sm-3 col-md-2'
-        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
-        self.helper.form_class = "form-horizontal"
+        from corehq.apps.domain.views.accounting import DomainSubscriptionView
         if on_annual_plan:
-            back_button_text = "Back to my Subscription"
-            urlname = DomainSubscriptionView.urlname
-        else:
-            back_button_text = "Select different plan"
-            urlname = SelectPlanView.urlname
-        self.helper.layout = crispy.Layout(
-            'name',
-            'company_name',
-            'message',
-            hqcrispy.FormActions(
-                hqcrispy.LinkButton(
-                    _(back_button_text),
-                    reverse(urlname, args=[self.domain]),
-                    css_class="btn btn-default"
-                ),
-                StrictButton(
-                    _("Submit"),
-                    type="submit",
-                    css_class="btn-primary",
-                ),
-            )
-        )
+            kwargs['back_button_text'] = "Back to my Subscription"
+            kwargs['back_button_url'] = DomainSubscriptionView.urlname
+
+        super().__init__(domain, web_user, data, *args, **kwargs)
 
     def send_message(self):
-        subject = "[Annual Plan Request] %s" % self.domain
-        context = {
-            'name': self.cleaned_data['name'],
-            'company': self.cleaned_data['company_name'],
-            'message': self.cleaned_data['message'],
-            'domain': self.domain,
-            'email': self.web_user.email
-        }
-        html_content = render_to_string('accounting/email/sales_request.html', context)
-        text_content = """
-        Email: %(email)s
-        Name: %(name)s
-        Company: %(company)s
-        Domain: %(domain)s
-        Message:
-        %(message)s
-        """ % context
-        send_html_email_async.delay(subject, settings.BILLING_EMAIL,
-                                    html_content, text_content,
-                                    email_from=settings.DEFAULT_FROM_EMAIL)
+        subject_tag = "[Annual Plan Request]"
+        super().send_message(subject_tag)
+
+
+class CustomPlanContactForm(PlanContactForm):
+
+    def __init__(self, domain, web_user, data=None, *args, **kwargs):
+        from corehq.apps.domain.views.accounting import DomainSubscriptionView
+        kwargs['back_button_text'] = "Back to my Subscription"
+        kwargs['back_button_url'] = DomainSubscriptionView.urlname
```

Comment:
Same as previous suggestion
```suggestion
        kwargs.setdefault('back_button_text', "Back to my Subscription")
        kwargs.setdefault('back_button_url', DomainSubscriptionView.urlname)
```